rstar/Cargo.toml

```diff
@@ -1,19 +1,17 @@
 [package]
 name = "rstar"
-version = "0.8.3"
-authors = ["Stefan Altmayer <stoeoef@gmail.com>"]
+version = "0.8.4"
```

Comment:
No big deal, but I usually don't bump the version until just before the release;

1. we don't know what other (breaking) changes might go in before release
2. less to keep track of so you don't have to double-check every time you make a PR if you should be bumping the version

Is there a benefit to bumping now?

Reply:
No, it was purely a "we'll be bumping anyway" thing. Will revert.